Lucas_attemptAtNewBranch/grabData.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+"""
+Lucas Kinsey
+20220309
+grabData File
+Quickly calls read mat file function and returns dictionary of a mouse
+mouse is specified based on the response of the user
+"""
+
+
+import BCI_analysis as bci
+
+def grabData():
+    ## Specify Mouse and load its data
+    mouseID = input("What is the Mouse's Name? ")
```

Comment:
Sorry for pointing this out again, but if I run
So generally I think there's two ways to handle arguments to a function.
1. If we know that the function would not be used directly from the commandline, i.e does not need arguments directly ffrom the user. In this case, its upto the one using that function in some other file on how to use it. For eg, we probably wont run `grabData.py` directly. This will most likely be used by some other function that will use `dataDict = grabData("BCI_26")`, therefore no need to use input() to get argument.
2. Sometimes we do need to use some scripts directly from the command line, in that case, the best way to handle arguments is to use argparse, https://docs.python.org/3/library/argparse.html. For eg we can use `python3 grabData.py --mouseID "BCI_26"`.